### PR TITLE
[INLONG-1085][Doc] Add documents for Airflow offline scheduler

### LINKED ISSUE #1074: [Improve][Doc] Fix incorrect expression in Offline Data Synchronization example
### Description

There are uncertainties in the Pulsar to MySQL example.
image
image

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### LINKED ISSUE #1085: [Feature][Doc] Add documents for Airflow offline scheduler
### Description

Add documentation for Airflow.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Zkplo**
Fixes #1085 

### Motivation

- Add documents for Airflow offline scheduler.
- Refactored the directory and classified the images.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
    - *Added integration tests for end-to-end deployment with large payloads (10MB)*
    - *Extended integration test for recovery after broker failure*

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md [hunk 1]
```diff
@@ -51,19 +51,19 @@ Password: inlong
 - 延迟时间：表示任务启动的延迟时间
 - 有效时间：包括起始时间和结束时间，调度任务只会在这个时间范围内执行
 
-![Create Offline Group](img/pulsar_mysql/normal_schedule.png)
+![Create Offline Group](img/pulsar_mysql/quartz/normal_schedule.png)
 
 crontab调度需要设置以下参数：
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md:56]:
```suggestion
crontab 调度需要设置以下参数：
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+![airflow_create_ynchronization_task](img/pulsar_mysql/airflow/airflow_create_ynchronization_task.png)
+
+### 创建数据流组
+![airflow_data_stream_group](img/pulsar_mysql/airflow/airflow_data_stream_group.png)
+
+### 创建数据源和数据目标
+该小节请参照: [Pulsar 到 MySQL 示例](./pulsar_mysql_example.md)
+
+### Airflow离线任务创建结果
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md:93]:
```suggestion
### Airflow 离线任务创建结果
```

Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md:93]: Please optimize this title.

File: docs/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+### Airflow REST API
+
+By default, Airflow will reject all REST API requests. Please refer to the [Airflow official documentation](https://airflow.apache.org/docs/apache-airflow-providers-fab/stable/auth-manager/api-authentication.html) for configuration.
+
+### Inlong Manager Configuration
+
+Modify the configuration file according to the configuration file requirements and restart Inlong Manager.
+```properties
+# Inlong Manager URL accessible by the scheduler
+schedule.engine.inlong.manager.url=http://192.168.101.2:8083
```
Comment (fuweng11) [inline docs/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md:66]: Do not provide a real address.

File: docs/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+# Management URL for Airflow
+schedule.engine.airflow.baseUrl=http://192.168.101.16:8080
```
Comment (fuweng11) [inline docs/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md:68]: Ditto.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+### Inlong Manager 配置
+
+根据配置文件要求，对配置文件进行修改，并重启 Inlong Manager 。
+
+```properties
+# Inlong Manager URL accessible by the scheduler
+schedule.engine.inlong.manager.url=http://192.168.101.2:8083
+# Management URL for Airflow
+schedule.engine.airflow.baseUrl=http://192.168.101.16:8080
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md:70]: Ditto.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+默认情况下，Airflow 会拒绝所有 REST API 请求。请参考[Airflow 官方文档](https://airflow.apache.org/docs/apache-airflow-providers-fab/stable/auth-manager/api-authentication.html)进行配置。
+
+### Inlong Manager 配置
+
+根据配置文件要求，对配置文件进行修改，并重启 Inlong Manager 。
+
+```properties
+# Inlong Manager URL accessible by the scheduler
+schedule.engine.inlong.manager.url=http://192.168.101.2:8083
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md:68]: Ditto.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md (hunk 1 above)
Comment (Zkplo) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md]: done.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md [hunk 7]
```diff
@@ -80,25 +80,25 @@ CREATE TABLE sink_table (
...
 
-![Create Source Fields](img/pulsar_mysql/source_field.png)
+![Create Source Fields](img/pulsar_mysql/quartz/source_field.png)
 
-![Create Sink Fields](img/pulsar_mysql/sink_field.png)
+![Create Sink Fields](img/pulsar_mysql/quartz/sink_field.png)
 
 ### 审批数据流
 
 页面点击【审批管理】->【我的审批】->【详情】->【通过】
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md:95]:
```suggestion
页面点击【审批管理】->【我的审批】->【详情】->【通过】。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md [hunk 8]
```diff
@@ -80,25 +80,25 @@ CREATE TABLE sink_table (
 
 数据目标中 点击 【新建】→【MySQL】，配置数据目标名称、库名和表名（test.sink_table）等信息。
 
-![Create Sink](img/pulsar_mysql/sink.png)
+![Create Sink](img/pulsar_mysql/quartz/sink.png)
 
 ### 配置字段信息
 
 分别在 【源字段】 和 【目标字段】中配置 Schema 映射信息，完成后点击 【提交审批】
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md:87]:
```suggestion
分别在 【源字段】 和 【目标字段】中配置 Schema 映射信息，完成后点击 【提交审批】。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md (hunk 7 above)
Comment (Zkplo) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md]: done.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md (hunk 8 above)
Comment (Zkplo) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/pulsar_mysql_example.md]: done.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+    name VARCHAR(255) NOT NULL,
+    create_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+```
+
+## Airflow 初始化
+
+### 获取初始 DAG
+
+它们可以在 [Inlong](https://github.com/apache/inlong) 获取。
```
Comment (fuweng11) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md:46]: What do they refer to? Please describe clearly

Comment (Zkplo) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/offline_data_sync/airflow_pulsar_mysql_example.md]: sure! done.

**Review by aloyszhang — APPROVED** [review]

**Review by fuweng11 — APPROVED** [review]

**Pull request merged**